test/suite/run.py

```diff
@@ -52,8 +52,11 @@
 # could pick the wrong one. We also need to account for the fact that there
 # may be an executable 'wt' file the build directory and a subordinate .libs
 # directory.
+env_builddir = os.getenv('WT_BUILDDIR')
 curdir = os.getcwd()
-if os.path.basename(curdir) == '.libs' and \
+if(env_builddir and os.path.isfile(os.path.join(env_builddir, 'wt'))):
```

Comment:
Small nit for python style.  The outer parentheses are not needed `if x and y:` rather than `if (x and y):`

Reply:
Fixed :)